docs/io.cozy.procedures.md

```diff
@@ -0,0 +1,31 @@
+[Table of contents](README.md#table-of-contents)
+
+# Cozy Procedures doctype
+
+## `io.cozy.procedures`
+
+The `io.cozy.procedures` doctype represents the administrative procedures.
+
+- `templateId`: {string} the id of the `io.cozy.procedures.templates` the procedure is related to.
```

Comment:
Since we're not having a template ATM we should remove this attribute ?

Reply:
What is a template ?